go.mod

```
@@ -132,7 +132,7 @@ require (
 	k8s.io/cri-api v1.20.11
 	k8s.io/klog v1.0.0
 	k8s.io/kubectl v1.20.11
```

It looks like we got out of sync at some point, but for housekeeping purposes we should bump the submodules ane keep them in the same 0.x.y format as upstream.  For example: https://github.com/k3s-io/k3s/blob/6bac01fc58b77f536f2018be84accb91972db95e/go.mod#L124-L135


```suggestion
	k8s.io/kubectl v0.20.14
```

---

yup